include/Engine/LowLevel/Vertex.hpp

```
@@ -10,8 +10,6 @@ struct Vertex
 	glm::vec3 normal = {0.f, 0.f, 1.f};
 
 	static constexpr GLsizei stride() noexcept {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)